bird.js

```diff
@@ -19,7 +22,13 @@ class Bird {
 
   show() {
     // draw the icon CENTERED around the X and Y coords of the bird object
-    image(this.icon, this.x - (this.width / 2), this.y - (this.height / 2), this.width, this.height);
+    image(
```

Comment:
My preferred style is to actually leave this in one line of code. (Don't hate me!)

Reply:
I'll adjust the eslintrc